src/org/labkey/test/components/DomainDesignerPage.java

```diff
@@ -26,24 +26,31 @@ public static DomainDesignerPage beginAt(WebDriverWrapper driver, String contain
     public DomainDesignerPage clickSave()
     {
         elementCache().saveButton.click();
-        String msg = waitForAnyAlert();
-        log("Clicking save.  Waited until alert with message ["+msg+"] appeared");
+
+        if (isAlertVisible())
+        {
+            String msg = waitForAnyAlert();
+            log("Clicking save.  Waited until alert with message [" + msg + "] appeared");
+        }
+        else
+           sleep(500); //TODO: wait for page load default
```

Comment:
We should definitely find a way to wait for a page load event, or at least wait for the url to change